snapcraft.yaml

```diff
@@ -197,6 +193,23 @@ parts:
         cp target/release/dump_syms $SNAPCRAFT_STAGE/usr/bin/
       fi
 
+  hunspell:
```

Comment:
Can you add a comment above this part to explain that this is a temporary stop-gap measure, and what the original implementation was, along with a link to [the bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1792006) ?

Reply:
Maybe also give credit to the original author of the hunspell-dictionaries content snap for this snippet?